lib/apiservers/service/restapi/handlers/vch_create.go

```diff
@@ -401,6 +401,7 @@ func buildCreate(op trace.Operation, d *data.Data, finder finder, vch *models.VC
 				c.OpsCredentials = common.OpsCredentials{
```

Comment:
Can we add some error handling here to reject requests if `vch.Endpoint.OperationsCredentials.GrantPerms` is set, but `vch.Endpoint.OperationsCredentials.User` is not?

This would match the CLI validation from 31358560bd91c7f2267f7f14dfab866e9e4f7934.

Reply:
In the CLI path, we make sure not do enable granting if no ops-user was specified (the CLI defaults to the admin user). Does that need to be done here?

Reply:
@lcastellano @zjs This is handled later in the function call to `c.OpsCredentials.ProcessOpsCredentials`. Do we want to handle this before it reaches the function call?